src/BloomBrowserUI/templates/customXMatter/MXBPamphlet-XMatter/MXBPamphlet-XMatter.less

```diff
@@ -25,25 +25,12 @@
 
 .insideBackCover {
     .marginBox {
-        .bloom-editable:only-child {
```

Comment:
The commit shows that this file was added by @gmartin7. The commit message gives a few clues as to what some of the code attempted to accomplish.
https://github.com/BloomBooks/Bloomdesktop/commit/dea23074478ffa3318ac6caedccd01842b15673a